src/components/UserPortal/UserSidebar/UserSidebar.tsx

```diff
@@ -6,6 +6,7 @@ import OrganizationsIcon from 'assets/svgs/organizations.svg?react';
 import SettingsIcon from 'assets/svgs/settings.svg?react';
 import TalawaLogo from 'assets/svgs/talawa.svg?react';
 import styles from './UserSidebar.module.css';
+import ProfileDropdown from 'components/ProfileDropdown/ProfileDropdown';
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. If this is the sidebar color, name it `--side-bar` , or whatever this section of code is defining. 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296